storm-webapp/src/main/java/org/apache/storm/daemon/drpc/DRPCServer.java

```diff
@@ -45,170 +46,180 @@
-            final String httpsKsType = (String) (conf.get(DaemonConfig.DRPC_HTTPS_KEYSTORE_TYPE));
-            final String httpsKeyPassword = (String) (conf.get(DaemonConfig.DRPC_HTTPS_KEY_PASSWORD));
-            final String httpsTsPath = (String) (conf.get(DaemonConfig.DRPC_HTTPS_TRUSTSTORE_PATH));
-            final String httpsTsPassword = (String) (conf.get(DaemonConfig.DRPC_HTTPS_TRUSTSTORE_PASSWORD));
-            final String httpsTsType = (String) (conf.get(DaemonConfig.DRPC_HTTPS_TRUSTSTORE_TYPE));
-            final Boolean httpsWantClientAuth = (Boolean) (conf.get(DaemonConfig.DRPC_HTTPS_WANT_CLIENT_AUTH));
-            final Boolean httpsNeedClientAuth = (Boolean) (conf.get(DaemonConfig.DRPC_HTTPS_NEED_CLIENT_AUTH));
-
-            //TODO a better way to do this would be great.
-            DRPCApplication.setup(drpc);
-            ret = UIHelpers.jettyCreateServer(drpcHttpPort, null, httpsPort);
-            
-            UIHelpers.configSsl(ret, httpsPort, httpsKsPath, httpsKsPassword, httpsKsType, httpsKeyPassword, httpsTsPath, httpsTsPassword, httpsTsType,
-                    httpsNeedClientAuth, httpsWantClientAuth);
-            
-            ServletContextHandler context = new ServletContextHandler(ServletContextHandler.NO_SESSIONS);
-            context.setContextPath("/");
-            ret.setHandler(context);
-
-            ServletHolder jerseyServlet = context.addServlet(ServletContainer.class, "/*");
-            jerseyServlet.setInitOrder(1);
-            jerseyServlet.setInitParameter("javax.ws.rs.Application", DRPCApplication.class.getName());
-            
-            UIHelpers.configFilters(context, filterConfigurations);
-            addRequestContextFilter(context, Config.DRPC_HTTP_CREDS_PLUGIN, conf);
-        }
-        return ret;
-    }
-    
-    private final DRPC _drpc;
-    private final ThriftServer _handlerServer;
-    private final ThriftServer _invokeServer;
-    private final Server _httpServer;
-    private Thread _handlerServerThread;
-    private boolean _closed = false;
-
-    public DRPCServer(Map<String, Object> conf) {
-        _drpc = new DRPC(conf);
-        DRPCThrift thrift = new DRPCThrift(_drpc);
-        _handlerServer = mkHandlerServer(thrift, ObjectReader.getInt(conf.get(Config.DRPC_PORT), null), conf);
-        _invokeServer = mkInvokeServer(thrift, ObjectReader.getInt(conf.get(Config.DRPC_INVOCATIONS_PORT), 3773), conf);
-        _httpServer = mkHttpServer(conf, _drpc);
-    }
+  private static final Logger LOG = LoggerFactory.getLogger(DRPCServer.class);
+  private final static Meter meterShutdownCalls = StormMetricsRegistry
+      .registerMeter("drpc:num-shutdown-calls");
+  private final DRPC _drpc;
+  private final ThriftServer _handlerServer;
+  private final ThriftServer _invokeServer;
+  private final Server _httpServer;
+  private Thread _handlerServerThread;
+  private boolean _closed = false;
+  public DRPCServer(Map<String, Object> conf) {
```

Comment:
This auto-generated formatting this is *SOOOOO* much less readable to me.  Is this because of our checkstyle settings?   Removing a newline that separates lines is just wrong, and having method calls with their parameters way to the left of the method names is no bueno.